packages/block-library/src/latest-posts/edit.js

```diff
@@ -148,6 +149,22 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 		]
 	);
 
+	// If a user clicks to a link prevent redirection and show a warning.
+	const { createWarningNotice, removeNotice } = useDispatch( noticeStore );
+	let noticeId;
+	const showRedirectionPreventedNotice = ( event ) => {
+		event.preventDefault();
+		// Always remove previous warning if any to show one at a time
+		// and use a unique id to announce the message in screen readers
+		// in every click.
+		removeNotice( noticeId );
+		noticeId = `block-library/core/latest-psts/redirection-prevented/${ uniqueId() }`;
+		createWarningNotice( __( 'Redirection is prevented in the editor.' ), {
```

Comment:
Is `showRedirectionPreventedNotice()` intended for use on non-links?

If not, non-technical users may find this more accessible:

> Links are disabled in the editor.

Reply:
It's not. I updated with your suggestion.